modules/random_cat_fact.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import json
+
+def random_cat_facts():
+    try:
+        url = 'https://cat-fact.herokuapp.com/facts/random'
+	response = request.get(url)
```

Comment:
NameError: name 'request' is not defined
```suggestion
	response = requests.get(url)
```